### PR TITLE
Enable testing of XRT board tools with ctest

### PR DESCRIPTION
**Pull request by sonals**

### CODE REVIEW

**Comment by sonals** [comment]
Produces a report like this:

```
Test project /home/sonals/git/XRT/build/Release
    Start 1: xbmgmt
1/4 Test #1: xbmgmt ...........................   Passed    0.18 sec
    Start 2: xbutil
2/4 Test #2: xbutil ...........................   Passed    0.17 sec
    Start 3: xbutil2
3/4 Test #3: xbutil2 ..........................   Passed    0.32 sec
    Start 4: xbmgmt2
4/4 Test #4: xbmgmt2 ..........................   Passed    0.18 sec

100% tests passed, 0 tests failed out of 4

Total Test time (real) =   0.86 sec
```

File: src/runtime_src/core/tools/xbutil2/SubCmdScan.cpp [hunk 1]
```diff
@@ -90,8 +90,11 @@ SubCmdScan::execute(const SubCmdOptions& _options) const
 
   // Walk the property tree and print info
   auto devices = pt.get_child_optional("devices");
-  if (!devices || (*devices).size()==0)
-    throw xrt_core::error("No devices found");
+  if (!devices || (*devices).size()==0) {
```
Comment (keryell) [inline src/runtime_src/core/tools/xbutil2/SubCmdScan.cpp:93]:
```suggestion
  if (!devices || devices->size()==0) {
```

File: src/runtime_src/core/tools/xbutil2/SubCmdScan.cpp [hunk 2]
```diff
@@ -90,8 +90,11 @@ SubCmdScan::execute(const SubCmdOptions& _options) const
...
+    // No devices when scanning is not an error condition
+    std::cout << "No devices found" << std::endl;
```
Comment (keryell) [inline src/runtime_src/core/tools/xbutil2/SubCmdScan.cpp:95]:
```suggestion
    std::cerr << "No devices found" << std::endl;
```

File: src/runtime_src/core/tools/xbutil2/SubCmdScan.cpp (hunk 1 above)
Comment (sonals) [reply src/runtime_src/core/tools/xbutil2/SubCmdScan.cpp]: updated

File: src/runtime_src/core/tools/xbutil2/SubCmdScan.cpp (hunk 2 above)
Comment (sonals) [reply src/runtime_src/core/tools/xbutil2/SubCmdScan.cpp:95]: yup, not an error condition

**Comment by chvamshi-xilinx** [comment]
retest this please

**Pull request merged**